rs/bitcoin/checker/src/main.rs

```diff
@@ -119,10 +119,15 @@ fn transform(raw: TransformArgs) -> HttpResponse {
 
 #[ic_cdk::init]
 fn init(arg: CheckArg) {
+    const DEFAULT_NUM_SUBNET_NODES: u16 = 13;
```

Comment:
Why is this the default? We could re-install the bitcoin checker canister from scratch no?

Reply:
I can add the num_subnet_nodes to InitArgs too, but to be backward compatible it needs to be optional, which is less than ideal. We could of course force it to be non-optional, which should work in this case because we just only installed it. Do you think it is okay to break backward compatibility now?

Reply:
>  break backward compatibility now

Generally, changing init/upgrade args doesn't count for backwards compatibility. I was mostly wondering why the default value is not `34`? I'm ok either way.

Reply:
I changed the default to 34. Will make a PR to fix the init arg later.